### PR TITLE
Fixes #569: The back arrow should go back to /issues when there's no history

### LINKED ISSUE #569: Back button doesn't work on the detail of an issue
1) go to https://webcompat.com/issues/734
2) click on back button

-> the back button does not work

### PR DESCRIPTION
**Pull request by miketaylr**
r? @magsout

### CODE REVIEW

**Comment by magsout** [comment]
I don't know why, but the code doesn't work

**Comment by miketaylr** [comment]
Hmm. @magsout can you do a mega-refresh (cmd + shift + r) to clear the cache and try again? I'm going to deploy to staging to test there as well, it was working on my machine last night.

**Comment by miketaylr** [comment]
Nope, not working on staging either. Let me try again @magsout.

**Comment by miketaylr** [comment]
OK, give it a test again @magsout. Also pushed this one to staging.

**Comment by miketaylr** [comment]
thanks!
